src/satellite_images_nso/api/nso_georegion.py

```diff
@@ -39,6 +40,30 @@ def correct_file_path(path):
     return path
 
 
+# Function to merge two GeoTIFF files
+@staticmethod
+def merge_tiffs(input_files, output_file):
+
+    raster_to_mosiac = []
+    # Open the input files
+    src_files_to_mosaic = [rasterio.open(file) for file in input_files]
+
+    # Merge the files
+    mosaic, out_trans = merge(src_files_to_mosaic)
+
+    output_meta = src_files_to_mosaic[0].meta.copy()
```

Comment:
Please copy the profile instead of meta (as these also contain the 'tiled' and 'interleave' properties, which are relevant for optimization!). See [this code](https://github.com/Provincie-Zuid-Holland/satellite_images_nso_extractor/blob/1d4c7967db988db7fc2b227b44e67ec7fb83a435/src/satellite_images_nso/_manipulation/nso_manipulator.py#L54) for example.

Reply:
Fixed